index.js

```diff
@@ -95,20 +95,28 @@ var API = require('./api-functions'),
     };
 
     /** The error callback for the Search API */
-    var errorHandler = function (err) {
+    var errorHandler = function (err)
+    {
```

Comment:
Hi @henhouse , could you please update your code to match the bracket style used? It's usually inline in JS and for consistency purposes I'd rather keep it that way. Thanks!